integration/templates/next-app-router/src/app/organizations-by-id/[id]/page.tsx

```diff
@@ -0,0 +1,18 @@
+import { auth } from '@clerk/nextjs/server';
+
+export default function Home({ params }): {} {
```

Comment:
What's the goal for shipping those pages within the app-router template?

Reply:
Exactly! For the integration tests, I needed an app that had:

- an org ID as a path token
- an org slug as a path token
- an org prefix that indicated the personal workspace


And I needed sub-pages under each of those. So here we are!